README.md

```diff
@@ -728,6 +729,30 @@ I will set the Heating setpoint of the node with id `4` located in the `office`
 
 `zwave/office/nodeID_4/thermostat_setpoint/heating`
 
+### Multicast
+
+You can send Multicast requests to _all values with a specific suffix_ of a _group_ of nodes in the network.
+
+Multicast API is accessible from:
+
+`<mqtt_prefix>/_CLIENTS/ZWAVE_GATEWAY-<mqtt_name>/multicast/<value_topic_suffix>/set`
+
+- `value_topic_suffix`: the suffix of the topic of the value I want to control using multicast.
+
+It works like the set value API without the node name and location properties.
+If the API is correctly called the same payload of the request will be published
+to the topic without `/set` suffix.
+
+Example of multicast command (gateway configured as `named topics`):
+
+`zwave/_CLIENTS/ZWAVE_GATEWAY-test/multicast/thermostat_setpoint/heating/set`
+
+Payload: `{nodes: [5, 7, 9], value: 25.5}`
```

Comment:
Need to check this

Reply:
> But when i send: {"nodes":[2,3],"value": 70.0}(Also value 70 (without the .0) is giving the same error)

THis is json, the other one was js format, I'm converting the payload from string to json so that's why it wasn't working with the first

Reply:
@ODINServ could you try with latest changes?

Reply:
It works with:
```{"nodes":[2,3],"value":99}```

But not with:
```{nodes:[2,3],value: 99}```

With error:
```  
z2m:Mqtt Message received on zwavejs2mqtt/_CLIENTS/ZWAVE_GATEWAY-server/multicast/switch_multilevel/endpoint_1/targetValue/set +34s
z2m:Gateway No nodes found in multicast request to switch_multilevel/endpoint_1/targetValue +0ms```

Reply:
@ODINServ like I said in my previous comment the not working format will never work, it's js non JSON :) now it picks the correct value at least

Reply:
Ah check, should the docs than contain the quotes as well ?

I've just tried with all 12 nodes, it works nice for now, i guess it will be even better with the real multicast!

Reply:
> i guess it will be even better with the real multicast!

Reply:
> Ah check, should the docs than contain the quotes as well ?

You right about docs, fixing them now

Reply:
I'll try a broadcast tomorrow, and see what happens.

Besides that i disabled all power reporting features of the nodes, i don't need that info.
I want to see if it makes sense if the network is less loaded that the unicast messages are send and received better.